### ISSUE TITLE
Selendroid 0.14 crashes on start

### ISSUE THREAD

**Issue by dimkin-eu** [selendroid/selendroid#813]
After #741 I managed to get Selendroid working with version 0.14

http://localhost:4444/wd/hub/status returning such info
```
{"value":{"os":{"name":"Windows 7","arch":"amd64","version":"6.1"},"build":{"browserName":"selendroid","version":"0.14.0"},"supportedDevices":[{"emulator":false,"screenSize":"(1080, 1920)","serial":"8fdf99cf","platformVersion":"19","model":"GT-I9515"}],"supportedApps":[{"mainActivity":"io.selendroid.androiddriver.WebViewActivity","appId":"io.selendroid.androiddriver:0.14.0","basePackage":"io.selendroid.androiddriver"}]},"status":0}
```

Server starting looks like

```
Feb 20, 2015 2:36:10 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf install -r C:\Users\dmunda\AppData\Local\Temp\selendroid-server-io.selendroid.androiddriver-0.14.0.apk
Feb 20, 2015 2:36:25 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->
pkg: /data/local/tmp/selendroid-server-io.selendroid.androiddriver-0.14.0.apk
Success
5191 KB/s (1435383 bytes in 0.270s)
<--
Feb 20, 2015 2:36:26 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf shell setprop log.tag.SELENDROID ERROR
Feb 20, 2015 2:36:26 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->

<--
Feb 20, 2015 2:36:26 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf shell am instrument -e main_activity io.selendroid.androiddriver.WebViewActivity -e server_port 8081 io.selendroid.io.selendroid.androiddriver/io.selendroid.server.ServerInstrumentation
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->

<--
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf forward tcp:8081 tcp:8081
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->
```

then it fails (note 0.13.0-SNAPSHOT mention in response ) an tries many times to restart webview @ device
```
<--
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.android.impl.AbstractDevice startLogging
INFO: starting logcat:
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.server.model.SelendroidStandaloneDriver waitForServerStart
INFO: Waiting for the Selendroid server to start.
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.android.impl.AbstractDevice isSelendroidRunning
INFO: Checking if the Selendroid server is running: http://localhost:8081/wd/hub/status
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] I/O exception (java.net.SocketException) caught when processing request: Connection reset
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] Retrying request
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] I/O exception (java.net.SocketException) caught when processing request: Connection reset
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] Retrying request
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] I/O exception (java.net.SocketException) caught when processing request: Connection reset
12:36:27 INFO  [nioEventLoopGroup-3-8] [org.apache.http.impl.execchain.RetryExec][][][][][] Retrying request
Feb 20, 2015 2:36:27 PM io.selendroid.standalone.android.impl.AbstractDevice isSelendroidRunning
INFO: Can't connect to Selendroid server, assuming it is not running.
Feb 20, 2015 2:36:29 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf shell echo $EXTERNAL_STORAGE
Feb 20, 2015 2:36:29 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->
/storage/emulated/legacy
<--
Feb 20, 2015 2:36:29 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf shell ls C:\storage\emulated\legacy/
Feb 20, 2015 2:36:29 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->
C:\storage\emulated\legacy/: No such file or directory
<--
Feb 20, 2015 2:36:29 PM io.selendroid.standalone.android.impl.AbstractDevice isSelendroidRunning
INFO: Checking if the Selendroid server is running: http://localhost:8081/wd/hub/status
Feb 20, 2015 2:36:30 PM io.selendroid.standalone.android.impl.AbstractDevice isSelendroidRunning
INFO: Got response status code: 200
Feb 20, 2015 2:36:30 PM io.selendroid.standalone.android.impl.AbstractDevice isSelendroidRunning
INFO: Got response value: {"value":{"supportedApps":[],"build":{"browserName":"selendroid","version":"0.13.0-SNAPSHOT"},"os":{"arch":"armeabi-v7a","version":"19","name":"Android"},"supportedDevices":[]},"status":0}
Feb 20, 2015 2:36:30 PM io.selendroid.standalone.server.model.SelendroidStandaloneDriver waitForServerStart
INFO: Selendroid server has started.
Feb 20, 2015 2:36:32 PM io.selendroid.standalone.server.model.SelendroidStandaloneDriver createNewTestSession
SEVERE: Error occurred while starting Selendroid session
org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.43.1', revision: '5163bceef1bc36d43f3dc0b83c88998168a363a0', time: '2014-09-10 09:43:55'
System info: host: 'QA121012N', ip: '10.30.24.95', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_11'
Driver info: driver.version: SelendroidStandaloneDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.window(RemoteWebDriver.java:882)
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.switchToWebView(SelendroidStandaloneDriver.java:320)
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.createNewTestSession(SelendroidStandaloneDriver.java:290)
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.createNewTestSession(SelendroidStandaloneDriver.java:210)
	at io.selendroid.standalone.server.handler.CreateSessionHandler.handleRequest(CreateSessionHandler.java:40)
	at io.selendroid.standalone.server.BaseSelendroidStandaloneHandler.handle(BaseSelendroidStandaloneHandler.java:45)
	at io.selendroid.standalone.server.SelendroidServlet.handleRequest(SelendroidServlet.java:131)
	at io.selendroid.server.common.BaseServlet.handleHttpRequest(BaseServlet.java:67)
	at io.selendroid.server.common.http.ServerHandler.channelRead(ServerHandler.java:53)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.traffic.AbstractTrafficShapingHandler.channelRead(AbstractTrafficShapingHandler.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:161)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:147)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:787)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:130)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:189)
	at java.net.SocketInputStream.read(SocketInputStream.java:121)
	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
	at sun.reflect.GeneratedMethodAccessor11.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.apache.http.impl.conn.CPoolProxy.invoke(CPoolProxy.java:138)
	at com.sun.proxy.$Proxy24.receiveResponseHeader(Unknown Source)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:253)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:194)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:85)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:186)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:233)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 30 more

Feb 20, 2015 2:36:32 PM io.selendroid.standalone.server.model.DeviceStore release
INFO: Releasing device HardwareDevice [serial=8fdf99cf, model=GT-I9515, targetVersion=ANDROID19]
Feb 20, 2015 2:36:32 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Executing shell command: C:\Users\dmunda\AppData\Local\Android\android-sdk\platform-tools\adb.exe -s 8fdf99cf shell am force-stop io.selendroid.androiddriver
Feb 20, 2015 2:36:33 PM io.selendroid.standalone.io.ShellCommand exec
INFO: Shell command output
-->

```

**Comment by dimkin-eu** [comment]
tried with other device
got such logcat http://pastebin.com/Wta7j9bZ

http://localhost:4444/wd/hub/status
```
{"value":{"os":{"name":"Windows 7","arch":"amd64","version":"6.1"},"build":{"browserName":"selendroid","version":"0.14.0"},"supportedDevices":[],"supportedApps":[{"mainActivity":"io.selendroid.androiddriver.WebViewActivity","appId":"io.selendroid.androiddriver:0.14.0","basePackage":"io.selendroid.androiddriver"}]},"status":0}
```

**Comment by hgc2002** [comment]
same thing here. I'm not a deep android guy but I several error messages like these:

Failed to resolve attribute 6
at android.content.res.TypedArray.getLayoutDimension
...

webviewchromiumpak --> Bluetooth permission required

could not write /storage/sdcard/appcrash.log
failed EACCES permission denied

and many more.

I'm using a brand new AVD machine based on Android 5.0.1, resolution 800x1280

**Comment by asm89** [comment]
I'm closing this issue due to inactivity. If you still experience the issue described on the latest release of selendroid feel free to file a new issue. If you do please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

**Comment by HackerBeaver** [comment]
good
